bokeh/sphinxext/bokeh_prop.py

```diff
@@ -43,24 +47,28 @@ class Bar(Model):
 
 PROP_NAMES = [
     name for name, cls in properties.__dict__.items()
-    if isinstance(cls, type) and issubclass(cls, properties.Property)
+    if isinstance(cls, type) and (issubclass(cls, properties.Property) or issubclass(cls, properties.PropertyFactory))
```

Comment:
@bryevdv, `Property` and `PropertyFactory` are unrelated, and `PropertyFactory` is used for e.g. `Include`, so at first I would say it makes sense. But, I'm not sure if at this point `Include` shouldn't have been already replaced with included properties. If so, then checking for `PropertyFactory` shouldn't be necessary.